openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -152,6 +154,56 @@ def test_get_one(self, mock_warning, _mock_info):
         self.assertEqual(actual_program, expected_program)
         self.assertFalse(mock_warning.called)
 
+    @mock.patch(UTILS_MODULE + '.get_programs')
+    @mock.patch(UTILS_MODULE + '.get_program_types')
+    def test_get_programs_with_type(self, mock_get_program_types, mock_get_programs, _mock_warning, _mock_info):
```

Comment:
@rlucioni TestGetProgramTypes specifically tests program types. The functionality of this particular test doesn't seem to be covered by that class. In catalog/utils.py we have two separate functions: get_program_types which just returns a dictionary of program types; and get_programs_with_type which returns the list of programs.